docs/operator-docs/collection-storage-best-practices.md

```diff
@@ -0,0 +1,191 @@
+---
+id: collection-storage-best-practices
+title: Collection Storage Best Practices
+hide_title: true
+---
+
+# Collection Storage Best Practices
+
+## Storage Configuration
+
+Cumulus allows for the configuration of many buckets for your files. Buckets are created and added to your deployment as part of the [deployment process](../deployment/deployment-readme#create-s3-buckets).
+
+Organizing your data is important for metrics you may want to collect. AWS S3 metrics are calculated at the bucket level, so it is easy to get metrics by bucket. You can get storage metrics at the key prefix level, but that is done through the CLI, which can be very slow for large buckets. It is very difficult to estimate costs at the prefix level.
```

Comment:
I would make this the first line of the doc, and add a few words.

```suggestion
Organizing your data is important for metrics you may want to collect. AWS S3 storage and cost metrics are calculated at the bucket level, so it is easy to get metrics by bucket. You can get storage metrics at the key prefix level, but that is done through the CLI, which can be very slow for large buckets. It is very difficult to estimate costs at the prefix level.
```

Reply:
Yeah I love this idea.